full-stack-practice/client/src/Catalog.tsx

```diff
@@ -1,8 +1,16 @@
 import { useEffect, useState } from 'react';
-import { Product } from '../../server/server.ts';
-import { toDollars } from '../src/lib/index';
+import { readCatalog, toDollars } from './lib';
 import { Link } from 'react-router-dom';
-import { ClientError } from '../../server/lib/client-error.ts';
+
+export type Product = {
```

Comment:
It would be better to put this in read.ts so it is in a location that every component can easily access.